client/gutenberg/extensions/giphy/edit.js

```diff
@@ -72,15 +71,12 @@ class GiphyEdit extends Component {
 		return split[ split.length - 1 ];
 	};
 
-	fetch = url => {
-		const { setAttributes } = this.props;
-		const xhr = new XMLHttpRequest();
-		xhr.open( 'GET', url );
-		xhr.onload = () => {
-			if ( xhr.status === 200 ) {
-				const res = JSON.parse( xhr.responseText );
-				const giphyData = res.data.length > 0 ? res.data[ 0 ] : res.data;
+	fetch = url =>
+		apiFetch( { url, credentials: 'omit', externalApi: true } ).then(
```

Comment:
Just to confirm, this will work in Jetpack wp-admin context because we're using `url` and not `path`?

(And `externalApi` isn't part of [apiFetch](https://github.com/WordPress/gutenberg/tree/5ff463a9a59e217a8de10eb88c140a91a3366947/packages/api-fetch).)

Some context from Slack:

> On its own, it is implemented as a generic fetch utility. But in a WordPress context and by using its “root URL” middleware, it’s assumed to prepend the WP REST API root to any paths it’s fed

[via](https://wordpress.slack.com/archives/C02QB2JS7/p1548167973056500)

Reply:
Looks like there's related convo going on in https://github.com/Automattic/wp-calypso/pull/30332#issuecomment-456704822